server/sonar-server/src/main/java/org/sonar/server/setting/ws/ResetAction.java

```diff
@@ -67,12 +68,14 @@ public ResetAction(DbClient dbClient, ComponentFinder componentFinder, SettingsU
   public void define(WebService.NewController context) {
     WebService.NewAction action = context.createAction("reset")
       .setDescription("Remove a setting value.<br>" +
+        "It's not possible to try to reset settings defined in sonar.properties.<br/>" +
         "Requires one of the following permissions: " +
         "<ul>" +
         "<li>'Administer System'</li>" +
         "<li>'Administer' rights on the specified component</li>" +
         "</ul>")
       .setSince("6.1")
+      .setChangelog(new Change("7.1", "Settings from sonar.properties are explicitly forbidden"))
```

Comment:
I suggest: "The settings defined in config/sonar.properties are read-only and can't be changed. HTTP status 400 is returned in this case."

Reply:
I agree with "The settings defined in config/sonar.properties are read-only and can't be changed", I'm not sure we should specify the HTTP 400.